src/climsight/climsight.py

```diff
@@ -286,25 +287,25 @@ def on_llm_new_token(self, token: str, **kwargs) -> None:
         # Climate Data
         st.markdown("**Climate data:**")
         st.markdown(
-            "Near surface temperature",
+            "Near surface temperature (in °C)",
         )
         st.line_chart(
             df,
             x="Month",
-            y=["Present Day Temperature", "Future Temperature"],
+            y=["Present Day T2m", "Future T2m"],
```

Comment:
I like temperature more, as the output is not for climatologist, for usual people.

Reply:
Renamed Temperature and Precipitation in latest commit.